### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by raspberri05**
issue #193. Added back the readme contents lost during v2 conversion. Please let me know if this if fine or if some things need to be removed. I currently just switched it back to the readme that had all the information.

### CODE REVIEW

**Review by IgnaceMaes — CHANGES_REQUESTED** [review]
Can the current README template be used, but with the description of the old one?

Some sections no longer apply:
- The old compatibility info should be replaced with the new one
- The old running instructions are not applicable anymore. The new ones can be found here: https://github.com/ember-learn/guidemaker-ember-template/blob/main/CONTRIBUTING.md.

**Comment by raspberri05** [comment]
Hi i've made the changes. Not sure why it didnt add my new commits to this PR so I opened another PR: https://github.com/ember-learn/guidemaker-ember-template/pull/197

Wasnt sure if you still wanted to releases section so I've kept it. thanks

**Comment by raspberri05** [comment]
Hi i made the changes and added the commits to this PR. Not sure if you wanted to put the releases section back but I did anyways, please let me know if further changes are needed. Thanks

File: README.md [hunk 1]
```diff
@@ -1,26 +1,73 @@
...
+See [guides-source](https://github.com/ember-learn/guides-source) for an example of how to use this addon.
 
 [Longer description of how to use the addon in apps.]
 
 ## Contributing
-
 See the [Contributing](CONTRIBUTING.md) guide for details.
 
+Releases
+------------------------------------------------------------------------------
```
Comment (mansona) [inline README.md:57]: we should not use this style of headers any more

**Comment by raspberri05** [comment]
I've made the requested changes. Please let me know if anything else needs to be changed before merging this.

**Comment by mansona** [comment]
@raspberri05 it looks like you deleted your fork so we're not able to merge this any more? 🤔 was that a mistake?

**Comment by driesdl** [comment]
@mansona shall i make a new PR with the changes above in it?

**Pull request closed without merging**